HOW_TO.md

```diff
@@ -103,23 +104,48 @@ Once the image is built, you can run it locally or push it to an image repositor
 Please note that there may be naming requirements for pushing images to a repository.
 
 **Testing the Docker image**
-Using the same image setup as used when [testing your algorithm](#test), a sample command line to run the image could be:
-```docker run --rm --mount "src=/user/myself,target=/mnt,type=bind" my_algorithm:latest --working_space "/mnt" "/mnt/images"```
+
+using the same image setup as used when testing your algorithm, a sample command line to run the image could be:
+
+```docker run --rm --mount "src=/user/myself,target=/mnt,type=bind" my_algorithm:latest --working_space "/mnt" --metadata "mnt/08f445ef-b8f9-421a-acf1-8b8c206c1bb8_metadata_cleaned.json "/mnt/images"```
 
 Breaking apart this command line, we have the following pieces:
 - `docker run` tells Docker to run an instance of the image (specified later in the command)
 - `--rm` tells Docker to remove the container (an image instance) when it's completed
 - `--mount "src=/user/myself,target=/mnt,type=bind"` specifies the */user/myself* path is to be made available as */mnt* in the container
 - `my_algorithm:latest` is the image to run (the running image is known as a *container*)
 - `--working_space "/mnt"` lets the software in the container know where its working disk space is located; files are created here
+- `--metadata "mnt/08f445ef-b8f9-421a-acf1-8b8c206c1bb8_metadata_cleaned.json"` specifies that the metadata file
+08f445ef-b8f9-421a-acf1-8b8c206c1bb8_metadata_cleaned.json will be made available to the container
 - `"/mnt/images"` specifies where the plot-level image files are located
 
 The `--mount` command line parameter is important since it allows the running container to access the local file system.
 The container can then load the images from the file system directly, without having to perform any copies.
-The parameters after the Docker image name are all relative to the target folder specified with this command line parameter. 
+The parameters after the Docker image name are all relative to the target folder specified with this command line parameter.
 
 Once the image files have been processed, the resulting CSV file(s) will be located in the folder at `/user/myself` (in this example).
 
+The result.json file should tell you what errors were found in the checks from testing.py (make sure to check the output in the CSV file(s) 
+even if the result.json file does not find errors)
+
+### (OPTIONAL) Testing the Docker container: 
```

Comment:
A better title might be: `(Optional) Testing your algorithm using a Docker container`